ci/PIPELINE.md

```diff
@@ -0,0 +1,117 @@
+## Install and configure a Tekton pipeline on Openshift
+
+1) Install Red Hat OpenShift Pipelines latest (1.9.2 at the time of this doc)  from the Operator Hub
+
+2) If you want interact via cli with the pipeline you can install locally Tekton cli 
+   to interact with the pipelines, you can download from the details of the operator installed (i.e. crc link) 
+   https://tkn-cli-serve-openshift-pipelines.apps-crc.testing/tkn/tkn-linux-amd64.tar.gz
+   The version proposed by the Operator is correctly aligned version with the tekton version.
+
+
+3) If isn't yet created create the project kogito-serverless-operator-system
+```sh
+oc new-project kogito-serverless-operator-system
+```
+
+4) Install the tekton task called kubernetes-actions
+```sh
+kubectl apply -f https://api.hub.tekton.dev/v1/resource/tekton/task/kubernetes-actions/0.2/raw
+```
+5) Apply the cluster_role and cluster_role_binding
+```sh
+kubectl create -f ci/role/cluster_role.yaml
+kubectl create -f ci/role/cluster_role_binding.yaml
+```
+
+6) Create the pipeline
+```sh
+kubectl apply -f ci/pipeline/kogito_serverless_operator_pipeline.yaml
+```
+
+7) Run the pipeline creating a pipeline run 
+```sh
+kubectl apply -f ci/pipeline/kogito_serverless_operator_pipeline_run.yaml
+```
+
+or with tekton cli
+
+or with tekton cli
+```sh
+tkn pipeline start kogito-serverless-operator-pipeline \
+-w name=shared-workspace,volumeClaimTemplateFile=https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/main/ci/volume/persistent_volume.yaml \
+-p deployment-name=kogito-serverless-operator \
+-p git-url=https://github.com/kiegroup/kogito-serverless-operator.git \
+-p git-revision=main \
+-p IMAGE='image-registry.openshift-image-registry.svc:5000/kogito-serverless-operator-system/kogito-serverless-operator:latest' \
+--use-param-defaults
+```
+
+8) Checks Pipeline execution
+ 
+Open the Pipeline Menu under the namespace/project kogito-serverless-operator-system
+or with tekton cli with the id printed starting the pipelinerun from the cli
+
+```sh
+tkn pipelinerun logs kogito-serverless-operator-pipeline-run-<id> -f -n <your-namespace>
+```
+
+### How to see the content of the workspace
+1) Create the task show_workspace_content
+```sh
+kubectl apply -f ci/task/show_workspace_content.yaml
+```
+
+2) Add the task show-workspace in the pipeline after the fetch-repository or build-image
+
+### How to deploy again 
+Go to the pipeline runs and ask on a already executed pipelinerun a rerun
```

Comment:
```suggestion
### How to deploy again 

Go to the pipeline runs and ask a rerun on an already executed pipeline
```

Reply:
btw, how to do that ?

Reply:
Is available a link on the pipelinerun executed

Reply:
on the UI you mean ?

Reply:
Yep

Reply:
maybe add a print as example/guidance?